File: src/appverify/AppVerifyClient.php
<?php

namespace telesign\enterprise\sdk\appverify;

use telesign\sdk\appverify\AppVerifyClient as _AppVerifyClient;

/**
 * App Verify is a secure, lightweight SDK that integrates a frictionless user verification process into existing
 * native mobile applications.
 */
class AppVerifyClient extends _AppVerifyClient {

  function __construct ($customer_id, $api_key, $rest_endpoint = "https://rest-ww.telesign.com", ...$other) {
    parent::__construct($customer_id, $api_key, $rest_endpoint, ...$other);
  }

}

File: src/main/java/com/telesign/enterprise/AppVerifyClient.java
package com.telesign.enterprise;

import java.net.Proxy;

public class AppVerifyClient extends com.telesign.AppVerifyClient {

    public AppVerifyClient(String customerId, String apiKey) {

        super(customerId, apiKey, "https://rest-ww.telesign.com");
    }

    public AppVerifyClient(String customerId, String apiKey, String restEndpoint) {
        super(customerId, apiKey, restEndpoint);
    }

    public AppVerifyClient(String customerId,
                         String apiKey,
                         String restEndpoint,
                         Integer connectTimeout,
                         Integer readTimeout,
                         Integer writeTimeout,
                         Proxy proxy,
                         final String proxyUsername,
                         final String proxyPassword) {
        super(customerId, apiKey, restEndpoint, connectTimeout, readTimeout, writeTimeout, proxy, proxyUsername, proxyPassword);
    }
}
